python/ct/client/tls_message.py

```diff
@@ -127,7 +127,9 @@ def read(self, message):
         """Read from the buffer into the protocol buffer message."""
         # TODO(ekasper): probably better not to modify the
         # original message until we're guaranteed to succeed?
-        for field in message.DESCRIPTOR.fields:
+        field_indices = message.DESCRIPTOR.fields_by_number.keys()
```

Comment:
Done, although field is referred enough times that I kept it (field = fields[i]).